tests/basic_setup/mkdocs.yml

```diff
@@ -3,4 +3,6 @@ use_directory_urls: true
 
 plugins:
     - search
-    - git-authors
\ No newline at end of file
+    - git-authors:
+          sort_authors_by: "contribution"
+          authorship_threshold_percent: 50
```

Comment:
This might break existing unit tests. Better to create a new, separate `mkdocs_w_contribution_sort_and_author_threshold.yml`.

I'd like to have a unit test for sorting by contribution. Not because the implementation is so complex or error prone, but to ensure the feature keeps working after future updates. Are you able to do this, or shall I pick it up?